apps/wing/src/commands/compile.test.ts

```diff
@@ -10,10 +10,24 @@ import { generateTmpDir } from "src/util";
 const exampleDir = resolve("../../examples/tests/valid");
 const exampleSmallDir = resolve("../../examples/tests/valid/subdir2");
 const exampleFilePath = join(exampleDir, "captures.test.w");
+const examplefilepath2 = join(exampleDir, "capture_primitives.test.w");
```

Comment:
Please follow the naming conventions used in the rest of the file

```suggestion
const exampleFilePath2 = join(exampleDir, "capture_primitives.test.w");
```

Reply:
Good catch, that was a typo.